cpp/src/DataStorm/TopicI.cpp

```diff
@@ -272,24 +302,24 @@ TopicI::getElementSpecs(int64_t topicId, const ElementInfoSeq& infos, const shar
 }
 
 void
-TopicI::attach(int64_t id, const shared_ptr<SessionI>& session, SessionPrx prx)
+TopicI::attach(int64_t id, shared_ptr<SessionI> session, SessionPrx peerSession)
 {
-    auto p = _listeners.find(ListenerKey{session});
```

Comment:
ListenerKey was not required for this map, we were using the default shared_ptr operator<